### PR TITLE
bugfix/393

### LINKED ISSUE #393: EDDT: Arrivals climb without bound
Inbound aircraft climb to `Infinity`. Two problems with that:
- they shouldn't be trying
- they shouldn't be able (performance)

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #393.

Prevent Aircraft from setting mcp altitude to infinity on spawn.

### CODE REVIEW

File: src/assets/scripts/client/aircraft/AircraftInstanceModel.js [hunk 1]
```diff
@@ -185,7 +185,11 @@ export default class AircraftInstanceModel {
 
         // TODO: There are better ways to ensure the autopilot is on for aircraft spawning inflight...
         if (options.category === FLIGHT_CATEGORY.ARRIVAL) {
-            const bottomAltitude = this.fms.getBottomAltitude();
+            let bottomAltitude = this.fms.getBottomAltitude();
+
+            if (bottomAltitude === Infinity) {
```
Comment (n8rzz) [inline src/assets/scripts/client/aircraft/AircraftInstanceModel.js]: this logic should happen in `fms.getBottomAltitude()`

Comment (erikquinn) [inline src/assets/scripts/client/aircraft/AircraftInstanceModel.js]: I agree, but we don't have access to the aircraft's altitude in that scope. Unless you think we should just pass that? :grimacing:

Comment (n8rzz) [inline src/assets/scripts/client/aircraft/AircraftInstanceModel.js]: no, you're not thinking 4th dimensionally

**Comment by erikquinn** [comment]
@n8rzz Right, but in the case that `minAltitudeFromLegs.length === 0`, we cannot return `0`, we must return the current altitude.

**Comment by n8rzz** [comment]
@erikquinn ahhhh, I smell what you're stepping in there.  I'll allow it 😉

**Pull request merged**